docs/attributes-registry/cicd.md

```diff
@@ -13,11 +13,26 @@ This group describes attributes specific to pipelines within a Continuous Integr
 | Attribute | Type | Description | Examples | Stability |
 |---|---|---|---|---|
 | <a id="cicd-pipeline-name" href="#cicd-pipeline-name">`cicd.pipeline.name`</a> | string | The human readable name of the pipeline within a CI/CD system. | `Build and Test`; `Lint`; `Deploy Go Project`; `deploy_to_environment` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| <a id="cicd-pipeline-result" href="#cicd-pipeline-result">`cicd.pipeline.result`</a> | string | The result of a pipeline run. | `success`; `failure`; `timeout` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
we could add skipped to the examples

```suggestion
| <a id="cicd-pipeline-result" href="#cicd-pipeline-result">`cicd.pipeline.result`</a> | string | The result of a pipeline run. | `success`; `failure`; `timeout`; `skipped` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Reply:
Added in a0443412ee94ab6c220c1a22247736b39e3e8fde